.docker/worker.dockerfile

```
@@ -97,6 +97,7 @@ RUN apk add --no-cache --virtual .build-deps \
     && pip3 install --no-cache-dir \
         PyExcelerate==0.6.7 \
         Pillow==8.1.* \
+    && pip install imagehash \
```

The Docker image has only Python 3 installed, so you only have to append imagehash to the list.

```suggestion
    && pip3 install --no-cache-dir \
        PyExcelerate==0.6.7 \
        Pillow==8.1.* \
        imagehash \
```